nav2_experimental_rl/random_crawl.py

```diff
@@ -0,0 +1,51 @@
+# Copyright (c) 2019 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os.path
+from turtlebot3_env import TurtlebotEnv
+import numpy as np
+import rclpy
+import parameters
+
+from rclpy.node import Node
+from time import sleep
+from keras.models import load_model
+
+def loadModel(env):
+    state = env.reset()
+    observation_space = len(state)
+    state = np.reshape(state, [1, observation_space])
+
+    project_path = os.path.abspath(os.path.dirname(__file__))
```

Comment:
I saw this use in the turtlebot demos, can we have this changed to use properly `rospkg` an example

```
rospack = rospkg.RosPack()
gazPath = rospack.get_path('turtlebot3_gazebo')
self.modelPath = gazPath + '/models/turtlebot3_square/goal_box/model.sdf'
```

Reply:
its cleaner and more clear if the install paths are wacky as python tends to get

Reply:
It seems like `get_package_share_directory` is  ROS2 equivalent of rospack.  https://answers.ros.org/question/288501/ros2-equivalent-of-rospackagegetpath/

Reply:
Fixed.